hddm/generate.py

```diff
@@ -409,6 +409,306 @@ def gen_rand_data(params=None, n_fast_outliers=0, n_slow_outliers=0, **kwargs):
 
     return data, subj_params
 
+def gen_rand_rlddm_pst_data(a,t,scaler,alpha,size=1,z=0.5,q_init=0.5,dual_alpha=0,subjs=1):
```

Comment:
I would rouhgly try to keep with the PEP8 style guide, e.g. spaces after commas and around math operators (but not around the `=`'s here)..

Reply:
You can also just run `autopep8` on these files.